src/main/java/fr/minuskube/inv/SmartInventory.java

```diff
@@ -191,6 +188,50 @@ public Builder manager(InventoryManager manager) {
             this.manager = manager;
             return this;
         }
+        
+        public String getId() {
+            return id;
+        }
+
+        public String getTitle() {
+            return title;
+        }
+
+        public InventoryType getType() {
+            return type;
+        }
+
+        public Optional<Integer> getRows() {
+            return rows;
+        }
+
+        public Optional<Integer> getColumns() {
+            return columns;
+        }
+
+        public boolean isCloseable() {
+            return closeable;
+        }
+
+        public int getUpdateFrequency() {
+            return updateFrequency;
+        }
+
+        public InventoryManager getManager() {
+            return manager;
+        }
+
+        public InventoryProvider getProvider() {
+            return provider;
+        }
+
+        public SmartInventory getParent() {
+            return parent;
+        }
+
+        public List<InventoryListener<? extends Event>> getListeners() {
+            return listeners;
```

Comment:
I'd suggest making the returned list immutable using
```java
com.google.common.collect.ImmutableList.copyOf()
```
or
```java
java.util.Collections.unmodifiableList()
```

Reply:
you are right